Sources/SwiftProtobuf/BinaryDelimited.swift

```diff
@@ -32,6 +32,21 @@ public enum BinaryDelimited {
     /// Messages are limited by the protobuf spec to 2GB; when decoding, if the
     /// length says the payload is over 2GB, this error is raised.
     case tooLarge
+
+    /// While attempting to read the length of a message on the stream, they
+    /// bytes were malformed for the protobuf format. This is different than
+    /// `BinaryDecodingError.malformedProtobuf` as that is raised for issues
+    /// within a message.
+    case malformedProtobuf
```

Comment:
This name makes it sound like a message is malformed. Maybe `malformedLength`?

Reply:
Works for me.